src/test/resources/org/broadinstitute/hellbender/tools/copynumber/model-segments-wes-tumor-2-allelic-counts-SM-74P4M-v1-chr20-downsampled.deduplicated.allelicCounts.tsv

```diff
@@ -0,0 +1,200 @@
+@HD	VN:1.5
+@SQ	SN:20	LN:1000000	UR:http://www.broadinstitute.org/ftp/pub/seq/references/Homo_sapiens_assembly19.fasta	AS:GRCh37	M5:0dec9660ec1efaaf33281c0d5ea2560f	SP:Homo Sapiens
+@RG	ID:GATKCopyNumber	SM:SM-74P4M-2
+CONTIG	POSITION	REF_COUNT	ALT_COUNT	REF_NUCLEOTIDE	ALT_NUCLEOTIDE
+20	61098	0	0	C	N
```

Comment:
I think this is because these sites are uncovered in this downsampled toy data. In any case, neither the REF nor ALT nucleotides are actually used anywhere---only their counts are.